core/api/src/jsonrpc/impl.rs

```diff
@@ -471,7 +471,6 @@ impl<Adapter: APIAdapter + 'static> AxonJsonRpcServer for JsonRpcImpl<Adapter> {
         Ok(all_logs)
     }
 
-
```

Comment:
Why do this?

Reply:
> Why do this?
i not modify the code ,just format the code.

Reply:
Sorry, I thought blank lines were removed, but it wasn't.